mkt/extensions/models.py

```diff
@@ -49,6 +49,9 @@ def pending(self):
         return self.filter(disabled=False, versions__deleted=False,
                            versions__status=STATUS_PENDING)
 
+    def pending_explicit(self):
```

Comment:
I think I prefer to call this one `pending()` and the other one `pending_versions()` or `with_pending_versions()`.